src/examples/java/com/amazonaws/crypto/examples/legacy/DiscoveryDecryptionExample.java

```diff
@@ -34,17 +35,11 @@
  */
 public class DiscoveryDecryptionExample {
 
+    private static final String ACCOUNT_ID = "658956600833";
```

Comment:
We should call out that this is an account that contains keys set up specifically for testing.

Reply:
Adding here and elsewhere.